core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/pagination/PaginationTag.java

```diff
@@ -41,11 +41,6 @@
   private String action;
   boolean actionIsJsFunction = false;
   private String pageParam;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "pageParam" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1068)